pkg/ddc/efc/transform_image_test.go

```diff
@@ -41,7 +41,7 @@ func TestParseFuseImage(t *testing.T) {
 func TestParseWorkerImage(t *testing.T) {
 	engine := &EFCEngine{}
 	image, tag, imagePullPolicy := engine.parseWorkerImage("", "", "")
-	if image != "registry.cn-zhangjiakou.aliyuncs.com/nascache/eac-worker" ||
+	if image != "registry.cn-zhangjiakou.aliyuncs.com/nascache/efc-worker" ||
 		tag != "update" || imagePullPolicy != "IfNotPresent" {
```

Comment:
Should change `tag != "update"` to `tag != "latest"` to pass the unit tests

Reply:
fix